code/modules/atmos_automation/statements.dm

```diff
@@ -37,77 +37,83 @@ var/global/automation_types=typesof(/datum/automation) - /datum/automation
 	return 0
 
 /datum/automation/proc/Export()
-	var/list/R = list("type"=type)
+	var/list/R = list("type" = type)
 
-	if(initial(label)!=label)
-		R["label"]=label
+	if(initial(label) != label)
+		R["label"] = label
 
-	if(initial(desc)!=desc)
-		R["desc"]=desc
+	if(initial(desc) != desc)
+		R["desc"]  = desc
 
-	if(children.len>0)
-		var/list/C=list()
+	if(children.len)
+		var/list/C = list()
 		for(var/datum/automation/A in children)
 			C += list(A.Export())
-		R["children"]=C
+
+		R["children"] = C
 
 	return R
 
 /datum/automation/proc/unpackChild(var/list/cData)
 	if(isnull(cData) || !("type" in cData))
 		return null
-	var/Atype=text2path(cData["type"])
+
+	var/Atype = text2path(cData["type"])
 	if(!(Atype in automation_types))
 		return null
+
 	var/datum/automation/A = new Atype(parent)
 	A.Import(cData)
 	return A
 
 /datum/automation/proc/unpackChildren(var/list/childList)
 	. = list()
-	if(childList.len>0)
+	if(childList.len > 0)
 		for(var/list/cData in childList)
 			if(isnull(cData) || !("type" in cData))
 				. += null
 				continue
-			var/Atype=text2path(cData["type"])
+
+			var/Atype = text2path(cData["type"])
 			if(!(Atype in automation_types))
 				continue
+
 			var/datum/automation/A = new Atype(parent)
 			A.Import(cData)
 			. += A
 
 /datum/automation/proc/packChildren(var/list/childList)
 	. = list()
-	if(childList.len>0)
+	if(childList.len > 0)
 		for(var/datum/automation/A in childList)
 			if(isnull(A) || !istype(A))
 				. += null
 				continue
+
 			. += list(A.Export())
 
 /datum/automation/proc/Import(var/list/json)
 	if("label" in json)
-		label = json["label"]
+		label    = json["label"]
 
 	if("desc" in json)
-		desc = json["desc"]
+		desc     = json["desc"]
 
 	if("children" in json)
 		children = unpackChildren(json["children"])
 
 /datum/automation/proc/fmtString(var/str)
-	if(str==null || str == "")
-		return "-----"
-	return str
+	return str || "-----"
 
-/datum/automation/Topic(href,href_list)
-	var/ghost_flags=0
+/datum/automation/Topic(var/href, var/list/href_list)
```

Comment:
yes but that's your opinion, so you're doing something based on your subjective opinion, even though it is technically objectively worse?